src/components/SpruceForm/FieldTemplates.tsx

```diff
@@ -150,7 +150,7 @@ const ArrayItem: React.VFC<
   return useExpandableCard ? (
     <StyledExpandableCard
```

Comment:
I am unable to test this, because I get an error when I try to save a subscription. 
<img width="469" alt="image" src="https://user-images.githubusercontent.com/13104717/178307337-217a3a81-1427-459d-bfe8-321ae5e10e9f.png">
tested [here](http://localhost:3000/project/sandbox/settings/notifications).

Reply:
Yeah i just merged it in